types/tailwindcss/resolveConfig.d.ts

```diff
@@ -0,0 +1,5 @@
+import { TailwindConfig } from './tailwind-config';
```

Comment:
```suggestion
import type { TailwindConfig } from './tailwind-config';
```
if you stick to 4.1, use type specific imports, as this is not a real object in JS.

Reply:
Thanks done